web/static/js/thesis-editor.js

```diff
@@ -56,25 +74,61 @@ class ThesisEditor extends React.Component {
   }
 
   pageDescription () {
-    const desc = this.descriptionMetaTag()
+    const desc = this.pageDescriptionMetaTag()
     return desc ? desc.content : null
   }
 
-  descriptionMetaTag () {
+  pageRedirectURL () {
+    return thesisContainer.getAttribute('data-redirect-url')
+  }
+
+  pageTemplate () {
+    return thesisContainer.getAttribute('data-template')
+  }
+
+  pageTemplates () {
+    return thesisContainer.getAttribute('data-templates').split(",")
```

Comment:
I'm not using jQuery, so I've learned a lot of cool vanilla JS features like `getAttribute`.